### ISSUE TITLE
Should not add fallback when there is already one, even named colors, or rgb()

### ISSUE THREAD

**Issue by WolfgangKluge** [postcss/postcss-color-rgba-fallback#11]
Following test

```css
.test1 {
    background-color: #fff;
    background-color: rgba(0, 0, 0, 0.5);
}

.test2 {
    background-color: orange;
    background-color: rgba(0, 0, 0, 0.5);
}

.test3 {
    background-color: rgb(128, 128, 128);
    background-color: rgba(0, 0, 0, 0.5);
}
```

leads to
```css

.test1 {
    background-color: #fff;
    background-color: rgba(0, 0, 0, 0.5);
}

.test2 {
    background-color: orange;
    background-color: #000000;
    background-color: rgba(0, 0, 0, 0.5);
}

.test3 {
    background-color: rgb(128, 128, 128);
    background-color: #000000;
    background-color: rgba(0, 0, 0, 0.5);
}
```

- :white_check_mark: `.test1` works 
- :bug: `.test2`
  named colors aren't recognized
- :bug: `.test3`
  One can argue that [there's a bug in IE6 & 7 with rgb()](https://css-tricks.com/ie-background-rgb-bug/), but even then, you should use the values from `rgb()`, not `rgba()`. So it should be `#808080` - and it should be before or insted of `rgb()`.

Edit: I removed two test cases with the `color`-function (from `postcss-color-function`) - they are redundant-

**Comment by MoOx** [comment]
It's a bug indeed, thanks for the test case.

**Comment by magsout** [comment]
Released as `1.3.0`
